### PR TITLE
Update sso-ldap.md

### PR DESCRIPTION
**Pull request by lindy65**
Added additional LDAP info from LB in [this doc](https://docs.google.com/document/d/1u7WY4IaF7Z9C9RSB9lZ16nsJhtR3RM3wJ6QVqNAzBhM/edit#bookmark=id.8ahd713507ub).

### CODE REVIEW

File: source/deployment/sso-ldap.md [hunk 1]
```diff
@@ -49,9 +49,35 @@ Notes:
...
 
-### Troubleshooting
+### Troubleshooting / FAQ
 
-The following are troubleshooting suggestions on common error messages and issues.
+The following are frequently asked questions and troubleshooting suggestions on common error messages and issues.
+
+##### When I first set up and synchronize LDAP, are the users automatically created in Mattermost? 
+
+No, the users are created on first log in.
```
Comment (jwilander) [inline source/deployment/sso-ldap.md]: This isn't very clear, maybe "No, each user is created on their first login."

File: source/deployment/sso-ldap.md [hunk 2]
```diff
@@ -49,9 +49,35 @@ Notes:
...
+No, the users are created on first log in. 
+
+##### If I want to add people to channels, can I pre-create users somehow? 
+
+Yes, using the [bulk import tool](https://docs.mattermost.com/deployment/bulk-loading.html#bulk-loading-data).
+
+##### How does deactivating users work? 
+
+- Deleting Users: Deleting a user from LDAP will deactivate them from Mattermost on the next sync. 
+- Disabling Users: Depends on the LDAP server. Some may deactivate the user in Mattermost, for others, such as Active Directory, you will need to specify with the user filter.
```
Comment (jwilander) [inline source/deployment/sso-ldap.md]: This doesn't have the tone of the rest of our docs, it sounds like a chat message response

File: source/deployment/sso-ldap.md [hunk 3]
```diff
@@ -49,9 +49,35 @@ Notes:
...
+
+##### If I want to add people to channels, can I pre-create users somehow? 
+
+Yes, using the [bulk import tool](https://docs.mattermost.com/deployment/bulk-loading.html#bulk-loading-data).
+
+##### How does deactivating users work? 
+
+- Deleting Users: Deleting a user from LDAP will deactivate them from Mattermost on the next sync. 
+- Disabling Users: Depends on the LDAP server. Some may deactivate the user in Mattermost, for others, such as Active Directory, you will need to specify with the user filter. 
+  - Active Directory, the qquery to filter out disabled users is: `(&(objectCategory=Person)(!(UserAccountControl:1.2.840.113556.1.4.803:=2)))`
```
Comment (jwilander) [inline source/deployment/sso-ldap.md]: `qquery` -> `query'

File: source/deployment/sso-ldap.md [hunk 4]
```diff
@@ -49,9 +49,35 @@ Notes:
...
+- Deselecting with user filter: Any users not selected with the user filter will be deactivated in Mattermost.
+
+##### Can I connect to multiple AD servers? 
+
+Not right now.  You'll need to connect the instances in a forest.
```
Comment (jwilander) [inline source/deployment/sso-ldap.md]: Tone is off here as well

File: source/deployment/sso-ldap.md (hunk 1 above)
Comment (lindy65) [reply source/deployment/sso-ldap.md]: Updated.

Comment (lindy65) [reply source/deployment/sso-ldap.md]: Updated.

File: source/deployment/sso-ldap.md (hunk 3 above)
Comment (lindy65) [reply source/deployment/sso-ldap.md]: Fixed

File: source/deployment/sso-ldap.md (hunk 2 above)
Comment (lindy65) [reply source/deployment/sso-ldap.md]: Updated

File: source/deployment/sso-ldap.md (hunk 4 above)
Comment (lindy65) [reply source/deployment/sso-ldap.md]: Not sure whether how I've updated is any better?

**Pull request merged**